test/unit/bootloader/config/grub2_test.py

```diff
@@ -777,7 +777,8 @@ def open_file(filename, mode=None):
                 'root=rootdev nomodeset console=ttyS0 console=tty0\n' \
                 'root=PARTUUID=xx'
             file_handle_grubenv.read.return_value = 'root=rootdev'
-            file_handle_menu.read.return_value = 'options foo bar'
+            file_handle_menu.read.return_value = \
+                'options foo\nlinux bullshit/boot/vmlinuz\ninitrd /boot/initrd'
```

Comment:
We probably shouldn't put profanity into the codebase...

Reply:
yes, however I have to admit it is hard not to get emotional when coding this :)

Reply:
Yeah, I agree. It's all terrible. :)

Reply:
ok, guys I agree, sorry I was a little stressed when I coded this. I will change it

Reply:
it's gone, hope that's better